Jellyfin.Api/Controllers/SyncPlayController.cs

```diff
@@ -69,6 +69,7 @@ public class SyncPlayController : BaseJellyfinApiController
         /// <returns>A <see cref="NoContentResult"/> indicating success.</returns>
         [HttpPost("Join")]
         [ProducesResponseType(StatusCodes.Status204NoContent)]
+        [Authorize(Policy = Policies.SyncPlayAccess)]
```

Comment:
It's already set on a controller level

Reply:
Oh, I see it now. :+1:

Reply:
Actually, this results in a different behaviour. What should happen to sessions that see their permissions revoked while participating in a group?

I was thinking that all request types should be allowed except for the ones that are explicitly marked now, otherwise that session won't be able to leave that group, ever.
Another approach would be to kick that session from the group, but I'm not aware of triggers when user settings are changed.

Reply:
True, there's no Event for that. Not ideal. But DefaultAuthorization should only apply to the leave group endpoint then?

Reply:
Is this a blocker, or something that can be easily fixed later without significant reorganization of the codebase?

Reply:
If it's the latter as I'm hoping, I'm gonna merge in about 1h so this can make 10.7.0.

Reply:
> True, there's no Event for that. Not ideal. But DefaultAuthorization should only apply to the leave group endpoint then?

It would be more restrictive but yeah, it can be a solution as well. I was seeing it more this way: if the user had the permission to join in the past then they might as well keep using the group they're in. In other words, I don't see the need to make the new restriction retroactive.

If a server admin wanted to kick an user from a group (after blocking the access to other/new groups) then a new administration page would fit better IMO.

> Is this a blocker, or something that can be easily fixed later without significant reorganization of the codebase?

Had to get some shuteye. A one liner should fix this.

Reply:
Policies can do whatever. If you really want to, you can make a policy that allows the user to call the group related endpoints if they are in that group

Reply:
> If you really want to, you can make a policy that allows the user to call the group related endpoints if they are in that group.

Ok, this one's better. I'll make a `Policies.SyncPlayInGroup` and use it where needed.

> I think we should just create an event when user policies are updated and then kick them.

And this can be a future addition, not strictly related to these policies.

Sounds good?